src/prototype_room_controller.js

```diff
@@ -116,5 +116,5 @@ Room.prototype.clearRoom = function() {
   _.each(constructionSites, (cs) => cs.remove());
 
   const creeps = this.findMyCreeps();
-  _.each(creeps, (cs) => cs.suicide());
+  _.each(creeps, (cs) => Creep.recycleCreep(cs));
```

Comment:
this can not work `_.each(structures, (s) => s.destroy());` so creep will not find a spawn in the room

Reply:
now it will search for other room.